pages/business/setup/signup.tsx

```diff
@@ -28,6 +30,10 @@ const LoginSchema = Yup.object().shape({
   confirmPassword: Yup.string()
     .required('Passwordwiederholung muss angegeben werden.')
     .oneOf([Yup.ref('password'), null], 'Passwörter stimmen nicht überein.'),
+  confirmContract: Yup.bool().oneOf(
+    [true],
+    'Sie müssen dem Vertrag zustimmen.'
+  ),
```

Comment:
This is blocking users who are not part of `isCareEnv` or `isHealthEnv` from registering as they don't have the `Checkbox` to confirm.